package.json

```diff
@@ -12,22 +12,14 @@
   "workspaces": {
     "packages": [
       "packages/*"
-    ],
-    "nohoist": [
-      "**/react-native",
-      "**/react-native/**"
     ]
   },
   "scripts": {
-    "app:android": "yarn workspace @golden-stack/app android",
-    "app:clear": "yarn workspace @golden-stack/app clear",
-    "app:ios": "yarn workspace @golden-stack/app ios",
-    "app:jetify": "yarn workspace @golden-stack/app jetify",
-    "app:pod": "yarn workspace @golden-stack/app pod:install",
-    "app:redirect-ports": "yarn workspace @golden-stack/app redirect-ports",
-    "app:relay": "yarn workspace @golden-stack/app relay",
-    "app:shake": "yarn workspace @golden-stack/app shake",
-    "app:start": "yarn workspace @golden-stack/app start",
+    "app:android": "yarn workspace @golden-stack/expo-app expo android",
+    "app:ios": "yarn workspace @golden-stack/expo-app expo ios",
+    "app:relay": "yarn workspace @golden-stack/expo-app relay",
+    "app:start": "yarn workspace @golden-stack/expo-app start",
+    "app:web": "yarn workspace @golden-stack/expo-app expo start -w",
```

Comment:
can you rename this to `@golden-stack/app`?

Reply:
of course

Reply:
resolved @jean-leonco